micrometer-core/src/main/java/io/micrometer/core/instrument/distribution/DistributionStatisticConfig.java

```diff
@@ -71,6 +72,25 @@ public class DistributionStatisticConfig implements Mergeable<DistributionStatis
     @Nullable
     private Integer bufferLength;
 
+    public DistributionStatisticConfig() {
+    }
+
+    public DistributionStatisticConfig(DistributionStatisticConfig original) {
```

Comment:
I'm not sure I getting why this is needed, should not this be the concern of `merge` instead?

Reply:
I thought it's the only way that that the subclass's constructor can copy all fields to a subclass instance, when merging